src/python/pants/backend/jvm/tasks/scalafmt.py

```diff
@@ -46,13 +47,13 @@ def source_extension(cls):
   def implementation_version(cls):
     return super(ScalaFmt, cls).implementation_version() + [('ScalaFmt', 5)]
 
-  def invoke_tool(self, _, target_sources):
+  def invoke_tool(self, _root, target_sources):
```

Comment:
Why? If you're going to name the param, it may as well have the same name as in the base class (`absolute_root`). Making it anonymous was presumably so that IDEs wouldn't complain about an unused param.

Reply:
`./pants lint` (either in pre-commit or on travis, I can't remember which) was complaining to me that the `_` in the list comprehension below was shadowing the `_` param name, and this showed up as an error. I was surprised.

Reply:
In that case let's use the original name? Introducing a third option is just going to add confusion...